samples/FormatFilterSample.Web/Startup.cs

```diff
@@ -41,7 +38,7 @@ public void Configure(IApplicationBuilder app)
                 routes.MapRoute("optionalroute",
                                 "{controller}/{action}.{format?}",
                                 new { controller = "Home", action = "Index" });
```

Comment:
Shouldn't we just use attribute routes for this? We don't ever recommend using conventional routes with APIs